code/data_structures/src/heaps/binomial_heap/README.md

```diff
@@ -0,0 +1,32 @@
+# BinomialHeap
```

Comment:
The README isn't just for C++, you know.

Reply:
Should I just remove it? Is it necessary to have?